### PR TITLE
fix empty log file "micrortps_bridge.log"

### PR DESCRIPTION
**Pull request by oneWayOut**
The log file **micrortps_bridge.log**   is empty if   make *_rtps  .
This PR is to fix the problem.

### CODE REVIEW

File: src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt [hunk 1]
```diff
@@ -82,7 +82,7 @@ if (NOT "${config_rtps_send_topics}" STREQUAL "" OR NOT "${config_rtps_receive_t
 			--agent-outdir ${micrortps_bridge_path}/micrortps_agent/src
 			--client-outdir ${micrortps_bridge_path}/micrortps_client
 			--idl-dir ${micrortps_bridge_path}/micrortps_agent/idl
-			>micrortps_bridge.log >/dev/null
+			>micrortps_bridge.log
```
Comment (TSC21) [inline src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt:85]:
```suggestion
			>micrortps_bridge.log 2>&1 || cat micrortps_bridge.log
```

**Review by TSC21 — CHANGES_REQUESTED** [review]
Thanks! I was trying to recall why /dev/null was added but I think it was a remaining of a development iteration. A more correct way of doing this though is to silent the output when it is successful and show it when it fails. Can you please apply my suggestion instead? Thank you!

**Comment by mrpollo** [comment]
@TSC21 I'm taking over the suggestion so we can merge this

**Comment by mrpollo** [comment]
@TSC21 This is now ready to merge, please go ahead :)

**Review by TSC21 — APPROVED** [review]

**Pull request merged**